js/storage.js

```diff
@@ -0,0 +1,22 @@
+const form1 = document.querySelector('#contact');
```

Comment:
- [ ] Kindly check if the form input values are getting persisted across refreshes. Currently, the data gets lost when I refresh the page. Please check the video below. I think this is happening because the form in the HTML file has the ID `form` but here you are trying to access a form with the ID `contact`.
Please update it accordingly. 🤝️

    <img style="max-width:300px;" src="https://cdn.loom.com/sessions/thumbnails/9e674fa892e841acb4525d3deae5583d-with-play.gif">